core/src/main/java/com/adobe/aem/dot/httpd/core/parser/HttpdConfigurationParser.java

```diff
@@ -74,7 +74,7 @@ public HttpdConfigurationParser(String repoPath) {
     ConfigurationViolations.clearViolations();
 
     List<ConfigurationLine> configLines = this.normalizeConfig(configFile,
-            configFile.getParentFile().getAbsolutePath(), true);
+            configFile.getParentFile().getPath(), true);
```

Comment:
Curious about the reason for this change. Will this affect the file location we report in each violation?

Reply:
From what I've read - this is an extreme absolute path.  What the 'real' difference is that it adds the Drive letter on Windows.